etcdserver/etcdserverpb/rpc.proto

```diff
@@ -289,6 +289,10 @@ message TxnResponse {
 // revision.
 message CompactionRequest {
   int64 revision = 1;
+  // physical is set so the RPC will wait until the compaction is physically
+  // applied to the local database such that compacted entries are totally
+  // removed from the backing store.
+  bool  physical = 2;
```

Comment:
there is an extra space between bool and physical? we do not align proto message.